app/modules/net.c

```diff
@@ -1088,18 +1088,14 @@ LROT_BEGIN(net_dns)
   LROT_FUNCENTRY( resolve, net_dns_static )
 LROT_END( net_dns, net_dns, 0 )
 
-LROT_BEGIN(net_if)
-  LROT_FUNCENTRY( info, net_if_info )
-LROT_END(net_if, net_if, 0)
-
 LROT_BEGIN(net)
   LROT_FUNCENTRY( createServer, net_createServer )
   LROT_FUNCENTRY( createConnection, net_createConnection )
   LROT_FUNCENTRY( createUDPSocket, net_createUDPSocket )
+  LROT_FUNCENTRY( ifinfo, net_iface_info )
```

Comment:
What is the idea behind exposing this as Lua **if**info but call it iface_info in C-land (rather than `net.ifinfo` -> `net_ifinfo`)? Do you expect `net_iface_newfunctionhere` in the future?

Reply:
I sort of do, but for the moment, consistency is probably better.  Can always adjust later.